static/js/jira-configuration.js

```diff
@@ -81,6 +81,7 @@ $("#cancel-backfill").click(() => {
 $(".sync-connection-link").click(event => {
 	const installationId = $(event.target).data("installation-id");
 	const jiraHost = $(event.target).data("jira-host");
+	const appId = $(event.target).data("app-id");
```

Comment:
Not sure how feasible to change this, but since `appId` means something else in the `GitHubServerApp`, I think it would be good to call it `gitHubAppId`. 

But /nit this one I guess.

Reply:
On the frontend, other places are already using `appId`, so followed the same name.
Leaving it as it is for now, the renaming can be done later on separately as a cleanup.